flintrock/flintrock.py

```diff
@@ -268,6 +274,8 @@ def launch(
         elif spark_git_commit:
             print("Warning: Building Spark takes a long time. "
                   "e.g. 15-20 minutes on an m3.xlarge instance on EC2.")
+            if spark_git_commit == "latest":
+                spark_git_commit = get_last_commit_sha(spark_git_repository)
```

Comment:
Here is a good place to tell the user what commit we're building Spark at, otherwise they will not know without logging into the cluster and running the appropriate git command.